nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -323,12 +323,14 @@ define([
         }
 
         /**
-         * This toggles which category (file type) should be shown
-         * // TODO: rename this and other widgets & elements to fileType
-         * @param {string} category - the category that should be shown
+         * This toggles which file type should be shown. This sets the
```

Comment:
minor semantic point but isn't it more the import app or import data type vs. file type? or have i been spending too much time in the parameter/file input section

Reply:
Well, now I'm confused. The design screenshots differentiate between FASTQ reads and SRA reads. Those are two different file types, but they both get converted into reads in the end. My feeling was that the categories in that column mapped to the different file types selected from the staging area.